index.bs

```diff
@@ -1184,64 +1184,97 @@ The {{MLOpSupportLimits}} has the following top level members, aside from these,
 <script type="idl">
 dictionary MLOpSupportLimits {
   MLInputOperandLayout preferredInputLayout;
-  MLSupportLimits input;
-  MLSupportLimits constant;
-  MLSupportLimits output;
+  [EnforceRange] unsigned long long maxTensorByteLength;
+  MLDataTypeLimits input;
+  MLDataTypeLimits constant;
+  MLDataTypeLimits output;
```

Comment:
(already merged, but looking back over it again) Would it make more sense for these to be `MLTensorLimits` to capture any maximum ranks [imposed by the API](https://github.com/webmachinelearning/webnn/issues/456#issuecomment-1692533722)? For example, DML has a global maximum rank of 8D (regardless of operator), and CoreML (Phillis can correct me if incorrect) has a limit of 5D.